### PR TITLE
Few enhancement

### PR DESCRIPTION
**Pull request by toitoinebzh**
## Problem
- Few warning of the script with package_check
- Occitan langage is not support
- No pull request template
- Wrong link in README

## Solution
- Correct script to reduce warning with package_check
- Add occitan support
- Add pull request template
- correct lnk in README

## PR Status
- [x] Code finished.
- [x] Tested with Package_check.

```
Package linter:                  SUCCESS
Installation:                    SUCCESS
Deleting:                        SUCCESS
Installation in a sub path:      SUCCESS
Deleting from a sub path:        SUCCESS
Installation on the root:        SUCCESS
Deleting from root:              SUCCESS
Upgrade:                         SUCCESS
Installation in private mode:    SUCCESS
Installation in public mode:     SUCCESS
Multi-instance installations:    SUCCESS
Malformed path:                  SUCCESS
Port already used:               SUCCESS
Backup:                          SUCCESS
Restore:                         SUCCESS
Change URL:                      SUCCESS
grep: /home/admin/package_check/levels.json: Aucun fichier ou dossier de ce type
Level of this application: 7 ()
	   Level 1: 1
	   Level 2: 1
	   Level 3: 1
	   Level 4: 1
	   Level 5: 1
	   Level 6: 1
	   Level 7: 1
	   Level 8: 0
	   Level 9: 0
	   Level 10: 0
You can find the complete log of these tests in /home/admin/package_check/Complete.log
Global working time for all tests: 11 minutes, 48 seconds.
Disable iptables rules.
Disable the network bridge.
```

- [x] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [x] Can be reviewed and tested.

## Validation
---
- [ ] **Code review** : 
- [ ] **Approval (LGTM)** :  

- **CI succeeded** : 
[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/pluxml_ynh%20PR34%20(antoine)/)

### CODE REVIEW

**Comment by kay0u** [comment]
Hi, are you planning to request the `high_quality` tag for this app? The PR template only exists for these applications. If not, can you at least remove the ping from the apps group, to avoid spamming this group?
If you want this tag, I'd be happy to review your app :-)

**Comment by toitoinebzh** [comment]
Hi @kay0u 
Thank you very much for your help
I am mainly looking for improving the quality of this app, high_quality tag is not really important
you are more than welcome to review it :-)

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+- [ ] Upgrade from last version tested.
+- [ ] Can be reviewed and tested.
+
+## Validation
+---
+- [ ] **Code review** : 
+- [ ] **Approval (LGTM)** :  
+
+- **CI succeeded** : 
+[![Build Status](https://ci-apps-hq.yunohost.org/jenkins/job/pluxml_ynh%20PR-NUM-/badge/icon)](https://ci-apps-hq.yunohost.org/jenkins/job/pluxml_ynh%20PR-NUM-/)
```
Comment (kay0u) [inline pull_request_template.md]:
```suggestion
[![Build Status](https://ci-apps-dev.yunohost.org/jenkins/job/pluxml_ynh%20PR-NUM-/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/pluxml_ynh%20PR-NUM-/)  
```

`high_quality` apps have a system to trigger a package_check automatically on PR, but if you don't ask for this tag, you must launch the package_check by yourself on https://ci-apps-dev.yunohost.org or on your side

**Comment by toitoinebzh** [comment]
Hi @kay0u ,
thank you for your help, I did not know this tool before
ti is running now, very useful
[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/pluxml_ynh%20PR34%20(antoine)/)

**Pull request merged**